scripts/email_utils.js

```diff
@@ -0,0 +1,38 @@
+import SESV2 from "aws-sdk/clients/sesv2";
+
+export async function sendAlarmEmail(subject, body) {
+  const ses = new SESV2({
+    apiVersion: "2019-09-27",
+    accessKeyId: process.env.AwsAccessId,
+    secretAccessKey: process.env.AwsSecretKey,
+    region: process.env.AwsSesEmailRegion,
+  });
+
+  // Split comma separated list of email recipients
+  const toAddr = process.env.AlarmEmailTo.split(",");
```

Comment:
Better to prefix `AlarmEmailTo` with a payout or something as we might add more scripts in the future that might need to send email to different parties and will need a different prefix.